demisto_sdk/commands/content_graph/strict_objects/classifier.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.strict_objects.common import (
+    DESCRIPTION_DYNAMIC_MODEL,
+    ID_DYNAMIC_MODEL,
+    NAME_DYNAMIC_MODEL,
+    BaseStrictModel,
+    create_model,
+)
+
+
+class _StrictClassifier(BaseStrictModel):
+    feed: Optional[bool] = None
+    incident_samples: Optional[List[str]] = Field(None, alias="incidentSamples")
+    indicator_samples: Optional[List[str]] = Field(None, alias="indicatorSamples")
+    propagation_labels: Optional[Any] = Field(None, alias="propagationLabels")
+    is_default: Optional[bool] = Field(None, alias="isDefault")
+    sort_values: Optional[Any] = Field(None, alias="sortValues")
+    id_: str = Field(..., alias="id")
+    version: int
```

Comment:
these two are repeated in more and more objects, try to see if we can consolidate models.